config/routes.rb

```diff
@@ -11,6 +11,9 @@
     resources :projects, only: [:create, :show, :index] do
       delete 'remove_dev', to: "projects#remove_dev"
       get 'payment', to: "projects#submit_payment"
+      post 'payment', to: "projects#pay"
+      patch 'complete', to: "projects#complete"
+      patch 'cancel', to: "projects#cancel"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.